neo4j/internal/db/connection.go

```diff
@@ -102,7 +102,7 @@ type Connection interface {
 	// Active streams will be discarded and the bookmark will be lost.
 	Reset(ctx context.Context)
 	// ForceReset is like Reset except it also sends a RESET messages for ready
-	// connections
+	// connections too
```

Comment:
```Go
	// ForceReset is like Reset except it also sends a RESET message for ready
	// connections, too.
```
Maybe? (punctuation and message in singular form)

Reply:
rephrased